instrumentation/grpc-1.5/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_5/client/GrpcClientTracer.java

```diff
@@ -23,9 +23,14 @@ protected GrpcClientTracer(Tracer tracer) {
     super(tracer);
   }
 
+  @Override
+  protected String getRpcSystem() {
+    return "grpc";
+  }
+
   public Span startSpan(String name) {
     SpanBuilder spanBuilder = tracer.spanBuilder(name).setSpanKind(CLIENT);
-    spanBuilder.setAttribute(SemanticAttributes.RPC_SYSTEM, "grpc");
+    spanBuilder.setAttribute(SemanticAttributes.RPC_SYSTEM, getRpcSystem());
```

Comment:
I'm guessing what @anuraaga meant in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2108#discussion_r564225768 is for this line of code to also live in the `RpcClientTracer` base class